package.json

```diff
@@ -98,10 +99,9 @@
   },
   "lint-staged": {
     "*.js": [
-      "prettier --write",
+      "prettier --write --trailing-comma es5 --print-width 120 --no-semi",
```

Comment:
There is no settings file: https://github.com/prettier/prettier/issues/62#issuecomment-294938907 - not really sure I agree with ^ - but for now we have to have them in our VS Code extension and in this function. Not a biggie.